### PR TITLE
Send sms campaign

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: src/Controller/Admin/AdminSmsCampaignCRUDController.php [hunk 1]
```diff
@@ -47,15 +50,15 @@ public function sendAction(SmsCampaign $smsCampaign, MessageBusInterface $bus):
             return $this->redirectToList();
         }
 
-        $paginator = $this->adherentRepository->findForAudience($smsCampaign->getAudience());
+        $paginator = $this->adherentRepository->findForSmsCampaign($smsCampaign);
 
         if ($paginator->getTotalItems() < 1) {
             $this->addFlash('sonata_flash_error', 'Cette campagne vise aucun contact');
```
Comment (Remg) [inline src/Controller/Admin/AdminSmsCampaignCRUDController.php:56]:
```suggestion
            $this->addFlash('sonata_flash_error', 'Cette campagne ne vise aucun contact.');
```

File: src/DataFixtures/ORM/LoadSmsCampaignData.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        $object->setStatus(SmsCampaignStatusEnum::ERROR);
+        $object->setResponsePayload('{"class":"Client::BadRequest","message":"Unknown body parameter sender"}');
+
+        $manager->flush();
+    }
+
+    public function getDependencies()
+    {
+        return [
+            LoadAdminData::class,
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadSmsCampaignData.php:64]: what is it for ?

File: translations/messages.fr.yml [hunk 3]
```diff
@@ -1160,6 +1160,7 @@ team.type.phoning: Phoning
 team_member_history.action.add: Ajout
 team_member_history.action.remove: Suppression
 
-sms_campaign.status.draft: Brouillon
-sms_campaign.status.sending: En cours d'envoi
-sms_campaign.status.done: Envoyée
+sms_campaign.status.draft: Brouillon 📝
+sms_campaign.status.sending: En cours d'envoi 💌
+sms_campaign.status.done: Envoyée ✅
+sms_campaign.status.error: Erreur ⛔️
```
Comment (Remg) [inline translations/messages.fr.yml:1166]: :muscle: :100: :ru:

**Review by Remg — APPROVED** [review]

**Comment by ottaviano** [comment]
[EM1289]

**Pull request merged**